networks/rpc/handler.go

```diff
@@ -476,7 +476,7 @@ func requestUpstream(ctx context.Context, msg *jsonrpcMessage, args []reflect.Va
 	}
 
 	rpcErrorResponsesCounter.Inc(1)
-	return msg.errorResponse(err)
+	return msg.errorResponse(fmt.Errorf("from upstream archive en: %w", err))
```

Comment:
```suggestion
	return msg.errorResponse(fmt.Errorf("from upstream rpc endpoint: %w", err))
```
While the intended use case is to point an archive node, there is no guarantee what kind of node the upstream is.

Reply:
Thanks.